lib/Jsrt/JsrtHelper.cpp

```diff
@@ -89,7 +89,7 @@ void JsrtCallbackState::ObjectBeforeCallectCallbackWrapper(JsObjectBeforeCollect
         pthread_key_create(&s_threadLocalDummy, DISPOSE_CHAKRA_CORE_THREAD);
 #endif
 
-#if defined(CHAKRA_STATIC_LIBRARY)
+#if defined(CHAKRA_STATIC_LIBRARY) || !defined(_WIN32)
```

Comment:
Are you ok with this workaround before you fix that? The current situation is all the initializations in this function are missing.

Reason I'm still working with shared lib is because static lib is unstable (hangs) and need more time. I need to ensure something working for coming events.